.github/CODEOWNERS

```
@@ -1,2 +1,2 @@
-#GUSINFO:MS Connectors - Infrastructure,MS VM Connector
-* @mulesoft/team-connectivity-infrastructure
+#GUSINFO:MS Connectors - Nexus,MS Connectors - Nexus, MS Connectors - Vector
```

```suggestion
#GUSINFO:MS Connectors - Nexus,MS VM Connector
@mulesoft/team-connectivity-vector 
```

Should be gus team and gus product tag
And the vector should be the only team owning the connector from now on